backend/app.js

```diff
@@ -24,11 +24,13 @@ app.use("/user", userRouter);
 app.use("/election", electionRouter);
 
 app.use((req, res) => {
-  res.status(404).json(
-    createApiResponse({
-      message: "Not Found",
-    }),
-    true,
+  res.status(404).send(
```

Comment:
aren't we sending json object here? why do you chose to use .send() instead of .json()?